### ISSUE TITLE
Spec: Time is not reported accurately

### ISSUE THREAD

**Issue by apainintheneck** [crystal-lang/crystal#14284]
## Bug Report

Basically the `crystal spec` command reports a time that seems unreasonably low. Maybe it's not counting build time and only runtime for the tests. If that's the case, I think only counting runtime is a mistake, since the tests always have to be built anyway.

```console
$ time crystal spec
..........

Finished in 4.76 milliseconds
10 examples, 0 failures, 0 errors, 0 pending

________________________________________________________
Executed in    6.72 secs    fish           external
   usr time    2.99 secs  191.00 micros    2.99 secs
   sys time    0.94 secs  594.00 micros    0.94 secs

$ crystal -v
Crystal 1.11.1 [0aa30372c] (2024-01-11)

LLVM: 15.0.7
Default target: x86_64-apple-macosx
```

**Comment by straight-shoota** [comment]
Yeah the reported time is the time it took to run the specs. Without upfront build time. I think this is fine and expected.

A spec program is just a regular Crystal program. And it can build and execute in different ways.
`crystal spec` is just a convenience feature of the compiler that combines the build and run steps into a single command.

If you want the entire time it takes to run `crystal spec`, using `time` as a separte command seems quite reasonable.

**Comment by apainintheneck** [comment]
If we showed both the build and run time for this command, I think it would be less confusing. I know at work the test suite for our Ruby monolith shows both file load time and spec runtime separately. That's sort of what I have in mind here. 

Other than that maybe the message can be improved somewhat. `Finished in X...` doesn't indicate what is being measured. Maybe `Finished running specs in X...` would make more sense here.
